recipes/spyder-vim/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "spyder-vim" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/spyder-vim-{{ version }}.tar.gz
+  sha256: 97c2ef31ae9bb24af1853d3741081cc27867f4c170e031a07daa213cc79577ca
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
```

```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps
```

To ensure nothing gets installed by `pip` aside from what `conda` installs

---

Just FYI the [official conda-forge docs](https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip) don't even suggest that :wink:

---

Hmm. That's what I see in various other recipes I maintain, but the provenance is unclear; perhaps CF used to recommend/require it, but no longer does? I assume you would know better than I...

---

I think it's changed over time. IIRC there are other protections against the problem you mentioned.